tests/pyincore-incubator/analyses/example/test_example.py

```diff
@@ -0,0 +1,23 @@
+from pyincore import IncoreClient
```

Comment:
Does this folder need to match the analysis folder and to be changed to use underscore "pyincore_incubator"?

Reply:
Good catch, let me fix that.